shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/files.py

```diff
@@ -230,14 +232,11 @@ def find_all_local_file_names(source_folder_name: str = None) -> List[str]:
     list: A list of all files that exist in the current working directory.
     """
     logger.debug(f"Finding all local file names in {source_folder_name}...")
+
     cwd = os.getcwd()
-    if not source_folder_name:
-        search_path = os.path.normpath(f"{os.getcwd()}/**")
-    else:
-        search_path = os.path.normpath(f"{cwd}/{source_folder_name}/**")
-    all_paths = glob.glob(search_path, recursive=True)
-    logger.debug(f"Found {len(all_paths)}.")
-    logger.debug(all_paths)
+    cwd_extension = os.path.normpath(f"{cwd}/{source_folder_name}/**")
```

Comment:
Just noting that this works as expected if `''` is provided but not when `None` is provided as the `source_folder_name`